src/diff/changes/nodeChanges.ts

```diff
@@ -14,6 +14,7 @@ import { RuleCategory } from "../ruleSet";
 export class NodeChanges {
   public added: { [key: string]: unknown };
   public removed: { [key: string]: unknown };
+  public ignored: { [key: string]: unknown };
```

Comment:
what is the purpose of ignored object? I don't see it being used anywhere

Reply:
Removed